### ISSUE TITLE
[a11y]: JAWS does not announce contents of ai-slug pop-up of the ai-chat component

### ISSUE THREAD

**Issue by Daniel-Schulz-Private** [carbon-design-system/carbon-labs#370]
### Browser

Firefox, Chrome

### Operating System

Windows

### Package version

0.14.1-canary.347

### React version

^1.66.0

### Automated testing tool and ruleset

JAWS

### Assistive technology

JAWS 2025

### Description

We
1. navigate to the `AI` button (using JAWS) of the header of the ai-chat component
2. open the pop-up window
3. switch focus to the pop-up window 
4. try to let JAWS announce the contents of the pop-up window

What we expect is that the contents is announced. We set very simple markup  as contents for the pop-up (a number of `<div>`, `<ol>`, `<li>`, `<h3>`, `<a>`, as well as  `<hr>` HTML elements). Visually, the contents is displayed correctly. The topmost `<div>` element is grouping the other elements.

What we get when we try to let JAWS announce the contents is the text `AI Explained - type a text`. 

I believe the first part `AI Explained` is the contents of our first `<div>` element of our contents. The second part `type a text`, stems from the fact, that the `ai-chat` component sets a role of `textbox`, which is considered editable and therefor leads to this announcement. Our contents is not editable though. May there be a more fitting role the ai-chat component may use here?

However, from here, whatever key or combination of keys we press, for example `ARROW down,` or `INSERT + ARROW down` does not announce the contents of our pop-up window.

### Reproduction/example

n/a

### Steps to reproduce

1. Leverage the ai-chat component
2. Specify the attribute `ai-slug-content` when creating the `<Chat>` component (use a `<div>` to group a few other `<div>`s with text contents as markup for the pop-up contents)
3. use JAWS to navigate to the slug pop-up window
4. use JAWS to announce the contents of the pop-up window

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-labs/blob/main/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon-labs/issues) for duplicate problems
